### DISCUSSION TITLE
nvidia-dkms vs nvidia-open-dkms

### DISCUSSION THREAD

**Discussion by UnconnectedBedna** [NVIDIA/open-gpu-kernel-modules#680] [Q&A]
I have tried to find information about this but fail every time so lets try asking here.

I have a 3090, currently running arch with nvidia-dkms drivers.

I have this really strange problem where the gpu power throttles at around 300w (both in mangohud and when running `nvidia-smi -q` when I game and it is happening) but the card has max power 350w.
Would installing the open kernel drivers fix that?

I am running AI:s locally. I was under the assumption cuda would not work if using open drivers, but I can not confirm if THIS version of drivers would work or not. Can someone clarify?

**Comment by mtijanic** [comment]
Hey there! As for version 555.xx, it is highly unlikely that you'll be seeing any meaningful difference between the open source and (default-configured) closed source driver versions. CUDA is certainly equally supported on both (and used in production on millions of machines).

As for your power usage, it might not actually be a bug. GPUs have a lot of different hardware units dedicated to different operations. If all the game is doing is pushing triangles, there's a portion of the chip that is not drawing _any_ power, and it's likely that the graphics bit is already saturated and can't really draw any more.

Can you paste the `nvidia-smi -q -d PERFORMANCE` output when you are in this state?

**Comment by UnconnectedBedna** [comment]
Thank you for the information. I'll stay on the "normal" dkms version of nvidia then. :)

<hr>

[nvidia-bug-report-nv_hdmi-game.log.gz](https://github.com/user-attachments/files/16378517/nvidia-bug-report-nv_hdmi-game.log.gz)

I have an old log file, not at the computer for a few days, but the behavior is still the same.
I had a game running at the same time, IIRC cities skylines 2.
I use prime-run if that matters (it is labeled hdmi, but I THINK the first version is when running with prime-run), but I have a logfile without using reverse prime, but it shows the exact same tendency.
[nvidia-bug-report-nv_hdmi.log.gz](https://github.com/user-attachments/files/16378576/nvidia-bug-report-nv_hdmi.log.gz)

I also found this log, that could be the reverse prime one, but it really should not matter, they all show the same power throttling IIRC.
[nvidia-bug-report.log.gz](https://github.com/user-attachments/files/16378605/nvidia-bug-report.log.gz)

The strange thing is when I use the card for AI calculations, it seems to have no problems going up to 350w.

I can run a new test in a few days when I am back at that computer.

I didn't actually read the logs now, so I might be completely wrong and one of them might be when I am NOT gaming, so take these with a grain of salt and wait until I am back and can do a fresh test with the 555 drivers.

**Edit**
I just read this: https://developer.nvidia.com/blog/nvidia-transitions-fully-towards-open-source-gpu-kernel-modules/
> For newer GPUs from the Turing, Ampere, Ada Lovelace, or Hopper architectures, NVIDIA recommends switching to the open-source GPU kernel modules.

Should I really not change to the open source modules with my 3090 (ampere)?
